rules/android_binary_internal/rule.bzl

```diff
@@ -52,6 +53,8 @@ def make_rule(
             "@bazel_tools//tools/jdk:toolchain_type",
         ] + additional_toolchains,
         _skylark_testable = True,
+        outputs = { 
```

Comment:
Isn't this only a valid change if `android_binary` outputs proguard mappings for every single `android_binary` target, which wouldn't be true for local debug APKs that don't run proguard.